jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -860,6 +803,8 @@ ecma_op_general_object_define_own_property (ecma_object_t *obj_p, /**< the objec
   return ecma_make_simple_value (ECMA_SIMPLE_VALUE_TRUE);
 } /* ecma_op_general_object_define_own_property */
 
+#undef ECMA_PROPERTY_TYPE_GENERIC
```

Comment:
This undef looks unnecessary. This isn't a header file and the undef is at the of the source, so won't be any redefinition of it. Did I missed something here?